As operator, fix workflow Verify Prod On-boarding

**Is your feature request related to a problem? Please describe.**

```
Error: Cannot find module '/home/runner/work/subscribie/subscribie/tests/browser-automated-tests-playwright/tests/test_prod_new_customer_can_sign_up.js
```
https://github.com/Subscribie/subscribie/actions/runs/9438898602/job/25996387119

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

--- a/.github/workflows/prod-test-onboarding-works.yml
+++ b/.github/workflows/prod-test-onboarding-works.yml
@@ -26,7 +26,7 @@ jobs:
           PLAYWRIGHT_HEADLESS: true
         run: |
           set -x
-          node --unhandled-rejections=strict tests/browser-automated-tests-playwright/tests/test_prod_new_customer_can_sign_up.js
+          node --unhandled-rejections=strict tests/browser-automated-tests-playwright/e2e/test_prod_new_customer_can_sign_up.js
       - uses: actions/upload-artifact@v2
         if: ${{ always() }}
         with:

